src/Lykke.Logs/LykkeLogToAzureSlackNotificationsManager.cs

```diff
@@ -42,25 +36,19 @@ public class LykkeLogToAzureSlackNotificationsManager : ProducerConsumer<LogEnti
             _slackNotificationsSender = slackNotificationsSender;
             _lastResortLog = lastResortLog ?? new LogToConsole();
             _component = AppEnvironment.Name;
-            _logLevels = new string[4]
-            {
-                LykkeLogToAzureStorage.ErrorType,
-                LykkeLogToAzureStorage.FatalErrorType,
-                LykkeLogToAzureStorage.WarningType,
-                LykkeLogToAzureStorage.MonitorType,
-            };
+            _logLevels = DefaultLogLevelsInit();
         }
 
         public LykkeLogToAzureSlackNotificationsManager(
             ISlackNotificationsSender slackNotificationsSender,
-            IEnumerable<string> logLevels,
+            HashSet<string> logLevels,
```

Comment:
ISet interface is better

Reply:
In ISet there is no clear Contains method. No clear advantage against Hashset,